### PR TITLE
Details entry

### PR DESCRIPTION
**Pull request by NadineGrosskreuz**
Added more detailed information (static data) and a hide and show button 
User Story #2

### CODE REVIEW

**Review by mbosselmann — COMMENTED** [review]
You rock! 🪨 Just some nit-picking remarks for you to consider. :) Keep it up!

File: src/components/UI/Accordion.styled.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+
+const Accordion = styled.button`
```
[the remark this answers is not in the document]
Comment (NadineGrosskreuz) [reply src/components/UI/Accordion.styled.js:3]: You are right, I changed it!

File: src/components/CardList/CardList.spec.js [hunk 2]
```diff
@@ -12,41 +12,40 @@ describe('CardList', () => {
...
-
-		expect(name[0]).toBeInTheDocument();
-		expect(name[1]).toBeInTheDocument();
-		expect(name[2]).toBeInTheDocument();
-
-		expect(address1).toBeInTheDocument();
-		expect(address2).toBeInTheDocument();
-		expect(address3).toBeInTheDocument();
+		const entryArray = screen.getAllByRole('listitem');
+		expect(entryArray.length).toBe(3);
```
[the remark this answers is not in the document]
Comment (NadineGrosskreuz) [reply src/components/CardList/CardList.spec.js:49]: I don't know why but the test doesn't work if I change it.

File: src/components/CardList/CardList.spec.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import '@testing-library/jest-dom';
 import CardList from './CardList';
 
 describe('CardList', () => {
-	it('renders three entries each with the category, the name and the address', () => {
+	it('renders three entries each with all the information', () => {
```
[the remark this answers is not in the document]
Comment (mbosselmann) [reply src/components/CardList/CardList.spec.js]: I think you can do it like you described. 👍

File: src/components/Card/Card.js [hunk 4]
```diff
@@ -1,12 +1,23 @@
...
-		<CardContainer>
+		<CardContainer onClick={handleOpen}>
 			<Typography variant="p">{category}</Typography>
 			<Typography variant="h2">{name}</Typography>
 			<Typography variant="p">{address}</Typography>
+			<Button>{show ? 'Weniger anzeigen' : 'Mehr anzeigen'}</Button>
+			{show && <Typography variant="p">{products}</Typography>}
+			{show && <Typography variant="p">{information}</Typography>}
+			{show && <Typography variant="p">{visited}</Typography>}
+			{show && <Typography variant="p">{rating}</Typography>}
```
Comment (doemser) [inline src/components/Card/Card.js:20]: Think about separating this four Typography Components into one seperate component, so you only need to 
`{show && <TheNewComponentYouWrote/>}` once to have the same effect.

**Review by doemser — CHANGES_REQUESTED** [review]
I just suggested a little component optimization.

File: src/components/Card/Card.js (hunk 4 above)
Comment (doemser) [reply src/components/Card/Card.js:20]: We already talked about this, I will approve your PR but maybe try rendering this information within a list in future.

**Review by doemser — APPROVED** [review]

File: src/components/Card/Card.js (hunk 4 above)
Comment (NadineGrosskreuz) [reply src/components/Card/Card.js:20]: Thanks! :)

**Pull request merged**